hack/tool/tackle

```diff
@@ -722,6 +725,65 @@ class TackleTool:
 
+    def migrateAssessments(self, assessmentsFileName="assessments", ignoreErrors=False):
+        cnt = 0
+        ## Export Pathfinder data for each Application
+        #for app in apiJSON(self.tackle2Url + "/hub/applications", self.tackle2Token):
+        #    for assm2 in apiJSON(self.tackle2Url + "/hub/pathfinder/assessments?applicationId=%d" % app.id, self.tackle2Token):
+        #        # Prepare Assessment and find its ID
+        #        assm               = Tackle2Object()
+        #        assm.id            = assm2['id']
+        #        assm.applicationId = assm2['applicationId']
+        #        assm.status        = assm2['status']
+        #        # Prepare Assessment questions and answers
+        #        asqa2 = apiJSON(self.tackle2Url + "/hub/pathfinder/assessments/%d" % assm.id, self.tackle2Token)
+        #        asqa               = Tackle2Object()
+        #        asqa.id                = asqa2['id']
+        #        asqa.applicationId     = asqa2['applicationId']
+        #        asqa.status            = asqa2['status']
+        #        asqa.stakeholders      = asqa2['stakeholders']
+        #        asqa.stakeholderGroups = asqa2['stakeholderGroups']
+        #        asqa.questionnaire     = asqa2['questionnaire']
+        #        self.add('assessments', asqa)
+        #saveJSON(os.path.join(self.dataDir, assessmentsFileName), self.data["assessments"])
+
+        # Upload assessment to Konveyor (expecting Pathfinder hard-coded questionnaire)
+        dictCollection = loadDump(os.path.join(self.dataDir, assessmentsFileName + '.json'))
+        print("Processing Pathfinder assessments..")
+        for passmnt in dictCollection:
+            print("# Assessment for Application %s" % passmnt["applicationId"])
+            # Skip if Assessment for given Application already exists
+            if len(apiJSON(self.tackle2Url + "/hub/applications/%d/assessments" % passmnt["applicationId"], self.tackle2Token, data={"questionnaire": {"id": 1}})) > 0:
+                print("  Assessment already exists, skipping.")
+                continue
+            # Create new assessment
+            assmnt = apiJSON(self.tackle2Url + "/hub/applications/%d/assessments" % passmnt["applicationId"], self.tackle2Token, data={"questionnaire": {"id": 1}}, method='POST')
+            for category in passmnt['questionnaire']['categories']:
+                print("Category %s %s" % (category["order"], category["title"]))
+                for question in category['questions']:
+                    print("  Question %s %s" % (question["order"], question["question"]))
+                    for option in question['options']:
+                        if option['checked'] == True:
+                            # Find corresponding option in newly created assessment and check it
+                            destSection = next(cat for cat in assmnt['sections'] if cat['order'] == category['order'])
```

Comment:
If you convert the old categories format to the new sections format by renaming the fields and adjusting the values as necessary, you can submit the converted sections without having to match up source and destination.